data-requests/pdr-issue-0013/pdr-issue-0013.sql

```diff
@@ -0,0 +1,38 @@
+WITH incentive_classes AS (
+
+    SELECT
+        parid,
+        taxyr,
+        user16
+    FROM iasworld.comdat
+    WHERE deactivat IS NULL AND cur = 'Y'
+    UNION
+    SELECT
+        parid,
+        taxyr,
+        user16
+    FROM iasworld.oby
+    WHERE deactivat IS NULL AND cur = 'Y'
+    UNION
+    SELECT
+        parid,
+        taxyr,
+        user16
+    FROM iasworld.dweldat
+    WHERE deactivat IS NULL AND cur = 'Y'
```

Comment:
**question (blocking):** Won't each of these return multiple card/lline records and thus not be distinct by PIN/year?

Reply:
I think that's fine - we probably don't want to exclude any card that could be an incentive class part of a parcel, even if the primary PIN isn't incentive class.